SPECS/bind/bind.spec

```diff
@@ -442,16 +440,15 @@ fi;
 %doc contrib/dlz/modules/sqlite3/testing/*
 
 %files libs
-%{_libdir}/*-%{version}*.so
+%{_libdir}/libdns*.so
+%{_libdir}/libisc*.so
+%{_libdir}/libns*.so
```

Comment:
Should these files include a version? something like `%{_libdir}/libdns*-%{version}*.so`

Reply:
I left the version off to catch the (unversioned) symlinks:
```
/usr/lib/libdns-9.20.0.so
/usr/lib/libdns.so
/usr/lib/libisc-9.20.0.so
/usr/lib/libisc.so
/usr/lib/libisccc-9.20.0.so
/usr/lib/libisccc.so
/usr/lib/libisccfg-9.20.0.so
/usr/lib/libisccfg.so
/usr/lib/libns-9.20.0.so
/usr/lib/libns.so
```